src/plugins/headers.ts

```diff
@@ -1,14 +1,15 @@
-import type { Argv } from 'yargs';
-import { AlphaCliConfig, AlphaCliArguments } from '../types';
+import type { Argv } from "yargs";
+import { AlphaCliConfig, AlphaCliArguments } from "../types";
 
 export const parseLine = (headers: Record<string, string>, line: string) => {
-  const [untrimmedKey, untrimmedValue] = line.split(':');
```

Comment:
This line fails whenever the header contains more than one colon `:`, such as when the header contains a JSON string (e.g. a `LifeOmic-Policy` header).

The fix is to split on the first `:`.